src/index.html

```diff
@@ -8,11 +8,49 @@
     />
     <title>Calendar</title>
     <link
-      rel="stylesheet"
-      href="styles/index.scss"
+      rel="styles"
+      href="styles/index.css"
     />
   </head>
+
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--length-31">
+        <div class="calendar__day calendar__day--start-sun">
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
+          <div class="calendar__day"></div>
```

Comment:
It's better to use a loop in your template engine or a scripting language to generate repetitive elements like calendar days. This will make your code more maintainable and less error-prone.